content/en/docs/Tasks/creating-a-catalog.md

```diff
@@ -78,21 +78,24 @@ EOF
 Is the third time the charm for `opm validate`?
 
 ```sh
-$ opm validate example-operator-index
+$ opm validate cool-catalog
 $ echo $?
 0
 ```
 
 Success! There were no errors and we got a `0` error code.
 
+#### Summary
 In the general case, adding a bundle involves three discreet steps:
-- Render the bundle into the index using `opm render <bundleImage>`.
+- Render the bundle into the Catalog using `opm render <bundleImage>`.
 - Add the bundle into desired channels and update the channels' upgrade edges
   to stitch the bundle into the correct place.
-- Validate the resulting index.
+- Validate the resulting `Catalog`.
```

Comment:
For consistency, I'd suggest dropping the backticks for code formatting, since we don't do this elsewhere for "catalog"